### DISCUSSION TITLE
Compatibility to Deployer V7

### DISCUSSION THREAD

**Discussion by mittererr** [sourcebroker/deployer-extended-database#12] [General]
Are there any plans to make this amazing tool compatible to Deployer V7?

THX

**Comment by kszymukowicz** [comment]
Thanks for kind words! 
I thought there is already official v7 but its still beta :)
I will integrate as soon as it will be stable. 
grtz

**Comment by kszymukowicz** [comment]
I forget to write. Version compatible with v7 was done like 4 months ago :)
